projects/packages/publicize/src/class-rest-controller.php

```diff
@@ -82,6 +92,18 @@ public function get_publicize_connections() {
 		return rest_ensure_response( $this->make_proper_response( $response ) );
 	}
 
+	/**
+	 * Gets the publicize shares count for the site.
+	 *
+	 * GET `jetpack/v4/publicize/shares-count`
+	 */
+	public function get_publicize_shares_count() {
+		$blog_id  = $this->get_blog_id();
+		$path     = sprintf( '/sites/%d/shares-count', absint( $blog_id ) );
+		$response = Client::wpcom_json_api_request_as_user( $path, '2', array(), null, 'wpcom' );
```

Comment:
As the share count will be for the blog, should we use `wpcom_json_api_request_as_blog` here?